content/en/docs/collector/management.md

```diff
@@ -82,13 +82,12 @@ Let's have a look at a concrete setup:
    the server-side OpAMP part and the collector (or a supervisor controlling the
    collector) implementing OpAMP client-side.
 
-You can try out a simple OpAMP setup yourself by using the [OpAMP protocol
+You can try out an OpAMP setup yourself by using the [OpAMP protocol
```

Comment:
Here the "simple" was more in the sense of "minimal" or "basic", since the setup does not provide a feature richt functionality. A different word may be a better choice.